.evergreen/run-load-balancer-tests.sh

```diff
@@ -74,11 +74,22 @@ echo $first
   --tests UnifiedTransactionsTest \
   --tests InitialDnsSeedlistDiscoveryTest
 second=$?
+echo $second
+
+./gradlew -PjdkHome=/opt/java/${JDK} \
+  -Dorg.mongodb.test.uri=${SINGLE_MONGOS_LB_URI} \
+  -Dorg.mongodb.test.transaction.uri=${MULTI_MONGOS_LB_URI} \
+  ${GRADLE_EXTRA_VARS} --stacktrace --info --continue driver-core:test \
+  --tests QueryBatchCursorFunctionalSpecification
```

Comment:
I was unable to add {{AsyncQueryBatchCursorFunctionalSpecification}} here because some tests fail with load balancer. Filed https://jira.mongodb.org/browse/JAVA-4259.